datahub-web-react/src/app/glossary/utils.ts

```diff
@@ -0,0 +1,23 @@
+import { EntityType } from '../../types.generated';
+import { GenericEntityProperties } from '../entity/shared/types';
+
+export const ROOT_NODES = 'rootNodes';
+export const ROOT_TERMS = 'rootTerms';
+
+export function getGlossaryRootToUpdate(entityType: EntityType) {
```

Comment:
Interesting how we have the special case values in this array. I didn't piece this together until i read all the code and took a step back. Just FYI